README.md

```diff
@@ -3,6 +3,11 @@ pillow-depends
 
 Pillow dependency sources -- cached for ci builds
 
+Pillow Wheels
+-------------
+
+These are used in the [Pillow OS X Wheel Builder](https://github.com/python-pillow/pillow-wheels)
```

Comment:
It does Linux wheels too, so "Pillow Wheel Builder".

Reply:
Okay, I've updated the commit.